packages/mobile/src/firebase/firebase.ts

```diff
@@ -293,6 +293,14 @@ function simpleReadChannel(key: string) {
   })
 }
 
+export async function readOnceFromFirebase(path: string) {
+  return firebase
+    .database()
+    .ref(path)
+    .once('value')
+    .then((snapshot) => snapshot.val())
```

Comment:
what does this section do? :o the snapshot part?

Reply:
When we make any query from Firebase records are returned on a snapshot.

A snapshot represents a node in the Firebase DB. The Firebase DB is organized similar to a json file, so for example given the following json:
```
{
  "example1": {
     "prop1": "value1",
     "prop2": "value2"
  },
  "example2": "hello"
}
```
If we query Firebase with the `example1` path the snapshot will contain the object found in that path which has the `prop1` and `prop2` fields. There are different ways of accessing these objects from the snapshot:
- If we call the `val()` function in the snapshot it will return a javascript object with all the contents of that node, no matter how many nested objects there are. This is the most common scenario.
- We can also call the `child()` function that returns a snapshot of whatever is in the path given. Por example, if we have a snapshot of `example1` we can call `snapshot.child('prop1')` which will return a snapshot of the `prop1` node. If we then call `val()` like `snapshot.child('prop1').val()` we'll get the value there which is `value1`. This is mostly useful to iterate on collections I think, we don't use it much.

Reply:
thank you Gonza! this makes a lot of sense :D